content/en/docs/tasks/debug-application-cluster/resource-usage-monitoring.md

```diff
@@ -20,99 +20,39 @@ where bottlenecks can be removed to improve overall performance.
 
 {{% capture body %}}
 
-In Kubernetes, application monitoring does not depend on a single monitoring
-solution. On new clusters, you can use two separate pipelines to collect
-monitoring statistics by default:
-
-- The [**resource metrics pipeline**](#resource-metrics-pipeline) provides a limited set of metrics related
-  to cluster components such as the HorizontalPodAutoscaler controller, as well
-  as the `kubectl top` utility. These metrics are collected by
-  [metrics-server](https://github.com/kubernetes-incubator/metrics-server)
-  and are exposed via the `metrics.k8s.io` API. `metrics-server` discovers
-  all nodes on the cluster and queries each node's [Kubelet](/docs/admin/kubelet)
-  for CPU and memory usage. The Kubelet fetches the data from
-  [cAdvisor](https://github.com/google/cadvisor). `metrics-server` is a
-  lightweight short-term in-memory store.
-  
-- A [**full metrics pipeline**](#full-metrics-pipelines), such as Prometheus, gives you access to richer
-  metrics. In addition, Kubernetes can respond to these metrics by automatically
-  scaling or adapting the cluster based on its current state, using mechanisms
-  such as the Horizontal Pod Autoscaler. The monitoring pipeline fetches
-  metrics from the Kubelet, and then exposes them to Kubernetes via an adapter
-  by implementing either the `custom.metrics.k8s.io` or
-  `external.metrics.k8s.io` API.
+In Kubernetes, application monitoring does not depend on a single monitoring solution. On new clusters, you can use [resource metrics](#resource-metrics-pipeline) or [full metrics](#full-metrics-pipeline) pipelines to collect monitoring statistics.
 
 ## Resource metrics pipeline
 
-### Kubelet
-
-The Kubelet acts as a bridge between the Kubernetes master and the nodes. It manages the pods and containers running on a machine. Kubelet translates each pod into its constituent containers and fetches individual container usage statistics from the container runtime, through the container runtime interface. For the legacy docker integration, it fetches this information from cAdvisor.  It then exposes the aggregated pod resource usage statistics through the kubelet resource metrics api.  This api is served at `/metrics/resource/v1alpha1` on the kubelet's authenticated and read-only ports.
-
-### cAdvisor
-
-cAdvisor is an open source container resource usage and performance analysis agent. It is purpose-built for containers and supports Docker containers natively. In Kubernetes, cAdvisor is integrated into the Kubelet binary. cAdvisor auto-discovers all containers in the machine and collects CPU, memory, filesystem, and network usage statistics. cAdvisor also provides the overall machine usage by analyzing the 'root' container on the machine.
```

Comment:
Should the paragraph about `cAdvisor` remain? Is `cAdvisor` integrated into the `kubelet`?

Reply:
@kbhawkey Yes, cAdvisor is integrated into the kubelet. However, since SIG Docs decided to remove all vendor-specific content, Zach asked me to remove that paragraph. cAdvisor is outside the CNCF pervue, hosted by Google.